shoop/core/pricing/__init__.py

```diff
@@ -157,13 +148,13 @@ def get_pricing_steps(self, context, product_id):
 
     def get_prices(self, context, product_ids, quantity=1):
```

Comment:
This API should also accept a list of IDs or a list of products (though the retval should still be as it is, a dict from ID to prices.